variables.tf

```diff
@@ -57,6 +57,12 @@ variable "file_triggers_enabled" {
   description = "Whether to filter runs based on the changed files in a VCS push"
 }
 
+variable "global_remote_state" {
+  type        = bool
+  default     = true
```

Comment:
I would suggest setting this to `null` by default. Especially since it influences the `remote_state_consumer_ids` variable (it's ignored when set to true).